runtime/gc_realtime/RealtimeGC.cpp

```diff
@@ -393,8 +225,7 @@ MM_RealtimeGC::workerSetupForGC(MM_EnvironmentBase *env)
 void
 MM_RealtimeGC::clearGCStats(MM_EnvironmentBase *env)
 {
-	_extensions->globalGCStats.clear();
```

Comment:
Why globalGCStats() is moved to delegate? Looks like it is called from OMR for Standard collectors.

Reply:
You're right, it can stay here instead of moving with java stats clear